api/graphql/reservation_units/reservation_unit_serializers.py

```diff
@@ -419,17 +490,131 @@ def validate_for_publish(self, data):
                 "minPersons can't be more than maxPersons"
             )
 
+        if not data.get("pricings", []):
+            raise serializers.ValidationError(
+                "pricings is required and must have one ACTIVE and one optional FUTURE pricing"
+            )
+
+    def validate_pricing_fields(self, data):
+        pricings = data.get("pricings", [])
+        if not pricings:
```

Comment:
Should we check that the pricings exists in some particular case?

Reply:
It is checked in `validate_for_publish`, but I'll do some refactoring and move the validation code under `reservation_unit_pricing_helper`. Flake8 started to complain about method being too complex 🤦